tests/Dapper.Tests/Dapper.Tests.csproj

```diff
@@ -38,4 +38,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' OR '$(TargetFramework)' == 'net5.0'">

```

Comment:
I don't like tying to the specific version here but I don't have a better idea at the moment. Suggestions welcome